util/patch/patch_test.go

```diff
@@ -122,27 +123,27 @@ var _ = Describe("Patch Helper", func() {
 				},
 			}
 
-			By("Creating a Node object")
-			Expect(testEnv.Create(ctx, obj)).ToNot(HaveOccurred())
-			key := client.ObjectKey{Name: obj.GetName()}
+			t.Log("Creating a Node object")
+			g.Expect(testEnv.Create(ctx, obj)).To(Succeed())
```

Comment:
The `g` that is referred here is initialised on line 112, which means that it's in the parent scope for this test. If this assertion fails it will fail not only this test but also any other's which rely on that `g`. 

Better to initialise the `g` inside the inner most `t.Run` so that if there's a failure, it only fails that particular subtest

Reply:
Good catch! I updated the PR to do so.